### PR TITLE
[IOPAE-236] Setup CI/CD

### PR DESCRIPTION
**Pull request by balanza**
#### List of Changes

* define code-review, labeler, and release pipelines
* configure Github identity

#### Motivation and Context

We configure a pipeline to perform code reviews on incoming changes in `apps` and/or `packages` folders. Also, a labeler pipeline is configured so that every pull request is automatically labelled according to code it impacts.

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/code-review.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  workflow_dispatch: {}
+
+jobs:
+  code_review:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```
Comment (pasqualedevita) [inline .github/workflows/code-review.yaml:26]:
```suggestion
        uses: actions/checkout@v3
```
better use hash

File: .github/workflows/code-review.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        with:
+          fetch-depth: 2
+
+      - name: Cache turbo build setup
+        uses: actions/cache@v3
```
Comment (pasqualedevita) [inline .github/workflows/code-review.yaml:31]:
```suggestion
        uses: actions/cache@v3
```
same, hash

File: .github/workflows/code-review.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        with:
+          path: .turbo
+          key: ${{ runner.os }}-turbo-${{ github.sha }}
```
Comment (pasqualedevita) [inline .github/workflows/code-review.yaml:34]: you can use yarn.lock file sha for better performance

File: .github/workflows/code-review.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+          restore-keys: |
+            ${{ runner.os }}-turbo-
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
```
Comment (pasqualedevita) [inline .github/workflows/code-review.yaml:39]:
```suggestion
        uses: actions/setup-node@v3
```
hash

File: .github/workflows/code-review.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    types: [opened, synchronize]
+    paths:
+      - apps/**
+      - packages/**
+  workflow_dispatch: {}
+
+jobs:
+  code_review:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
```
Comment (pasqualedevita) [inline .github/workflows/code-review.yaml:22]:
```suggestion
    runs-on: ubuntu-22.04
```

**Review by pasqualedevita — DISMISSED** [review]

File: .github/workflows/code-review.yaml (hunk 1 above)
Comment (balanza) [reply .github/workflows/code-review.yaml:26]: You mean something like `actions/checkout@1f9a0c22da41e6ebfa534300ef656657ea2c6707`?

Comment (pasqualedevita) [reply .github/workflows/code-review.yaml:26]: correct

**Comment by balanza** [comment]
@pasqualedevita I'd rather pin actions to a semver instead of a commit sha. What d'you think?

**Review by rodoviti — APPROVED** [review]
lgtm

**Pull request merged**